daisy/common.go

```diff
@@ -280,3 +280,26 @@ func CombineGuestOSFeaturesBeta(features1 []*computeBeta.GuestOsFeature,
 	})
 	return ret
 }
+
+// UpdateInstanceNoExternalIP updates Create Instance step to operate
+// when no external IP access is allowed by the VPC Daisy workflow is running in.
+func UpdateInstanceNoExternalIP(step *Step) {
+	if step.CreateInstances != nil {
+		for _, instance := range step.CreateInstances.Instances {
+			if instance.Instance.NetworkInterfaces == nil {
+				return
```

Comment:
why return instead of continue?

Reply:
should be continue. I copied from daisy_utils. Changed there as well.